<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 192, 255);
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-30</y>
      <width>701</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 255, 191);
background-color: rgb(234, 255, 244);
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);


</string>
      </property>
      <property name="text">
       <string>MENU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>Ejercicio 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salir">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="btn_back1">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>Regresar a Menú</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Elija un número para determinar la cantidad de filas de un triángulo</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_filas">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_triangulo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>591</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="btn_back1_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>Regresar a Menú</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tarjeta">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202, 255, 244);
font: 10pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_valid">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(79, 79, 79);
color: rgb(255, 255, 255);
font: 11pt &quot;Sitka&quot;;


</string>
      </property>
      <property name="text">
       <string>Validar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ingrese su número de tarjeta de crédito para validar si es real</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Elija una de las siguientes conversiones</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_conversiones">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Satki&quot;;
background-color: rgb(126, 234, 255);
border-color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>Letra a Binario</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Letra a Decimal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Letra a Hexadecimal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Binario a Letra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Decimal a Letra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hexadecimal a Letra</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lE_convertir">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(47, 47, 47);</string>
      </property>
      <property name="text">
       <string>Ingrese lo que desea convertir:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back1_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>Regresar a Menú</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_convertir">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(79, 79, 79);
color: rgb(255, 255, 255);
font: 11pt &quot;Sitka&quot;;


</string>
      </property>
      <property name="text">
       <string>Convertir</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_conversion">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Sitka&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
